ecommerce/src/components/main_page/MainPage.tsx

```diff
@@ -1,7 +1,83 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import MainPageBackGround from '../../images/backgrounds/mainBackGround2.jpg';
+import Rose1 from '../../images/icon/rose1.png';
+import Rose2 from '../../images/icon/rose2.png';
+import Rose3 from '../../images/icon/rose3.png';
+import Rose4 from '../../images/icon/rose4.png';
+import Rose5 from '../../images/icon/rose5.png';
+import Rose6 from '../../images/icon/rose6.png';
+import Rose7 from '../../images/icon/rose7.png';
+import Rose8 from '../../images/icon/rose8.png';
+import Rose9 from '../../images/icon/rose9.png';
+import Rose10 from '../../images/icon/rose10.png';
+import { getDiscountCode } from '../catalog_page/requests';
```

Comment:
it would be better to use one image with different sizes ( scale or something like this)